core/cli/src/plugin.ts

```diff
@@ -13,7 +13,7 @@ import {
 import resolveFrom from 'resolve-from'
 import type { Logger } from 'winston'
 import { EntryPoint, PluginOptions, RawConfig, ValidPluginsConfig } from './config'
-import { Conflict, isConflict } from './conflict'
+import { Conflict, isConflict } from '@dotcom-tool-kit/types/src/conflict'
```

Comment:
**question:** do we want to import from a compiled version of the file rather than from `src`?